BEST_PRACTICES.md

```diff
@@ -0,0 +1,88 @@
+# Introduction
```

Comment:
Editors support automatic line breaks now, so we don't need to do this manually. The major problem with this is the diffability once you start breaking sentences differently into lines. I prefer one line per sentence for this reason.